src/transactions/index.js

```diff
@@ -21,11 +24,21 @@ export default function TransactionPage() {
       <div className="row mb-5 mt-4">
         <div className="col">
           <h3 className="mt-1">Transactions</h3>
+          <p>
+            Category Id from transaction filters:
+            {filter.categoryId}
+          </p>
+
```

Comment:
This paragraph element is helpful for testing, but we can take it out now that we're getting ready to actually hook it up with the TransactionTable